content/docs/bdd/overview.md

```diff
@@ -1,90 +0,0 @@
----
-title: BDD Overview
-subtitle: Basic principles and practices of BDD
----
-
-Behaviour-Driven Development (BDD) is a set of practices that aim to reduce
-some common wasteful activities in software development:
-
-* Rework caused by misunderstood or vague requirements
-* Technical debt caused by reluctance to refactor code
-* Slow feedback cycles caused by silos and hand-overs
-
-BDD aims to narrow the communication gaps between team members, foster better understanding of
-the customer and promote continuous communication with real world *examples*.
-
-Examples describe how the software is *intended to behave*, often illustrating a particular business rule or requirement.
-
-{{% note "A simple example" %}}
-> Liz should be asked to guess again when she guesses "joke"
-
-This example is from a word guessing game. It's illustrating a rule
-stipulating that a guess must be 5 letters.
-{{% /note %}}
-
-BDD can be split in two parts - *Deliberate Discovery* and *Test-Driven Development*.
-
-# Deliberate Discovery
-
-There are many reasons why software projects go wrong. A very common reason
-is that different people in the organisation or on the team have very different
-understandings of how the software should behave, and what problems it's trying
-to solve.
-
-> Ignorance is the single greatest impediment to throughput. - [Dan North](https://dannorth.net/2010/08/30/introducing-deliberate-discovery/)
-
-Teams that *deliberately* seek to discover what they are ignorant about *before*
-development starts are more productive, because there is less rework.
-
-The most effective way to do this is through conversation and collaboration between
-key stakeholders, such as:
-
-* Product owners
-* Business analysts
-* Domain experts
-* Users
-* Programmers
-* UX designers
-* Testers
-* Ops engineers
-* And probably some others
-
-The Cucumber way to do this is [Example Mapping](https://cucumber.io/blog/2015/12/08/example-mapping-introduction), a
```

Comment:
Section on / link to example mapping would still be useful in the Discovery phase?

Reply:
Yes, I plan to make a whole new section on Discovery and the Example Mapping stuff will be in it.